<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFind</class>
 <widget class="QDialog" name="DialogFind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find text</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Text to find</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editText"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkCaseSensitive">
     <property name="text">
      <string>C&amp;ase sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkWholeWords">
     <property name="text">
      <string>&amp;Whole words only</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkFindBackward">
     <property name="text">
      <string>Find &amp;backward</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkCaseSensitive</tabstop>
  <tabstop>checkWholeWords</tabstop>
  <tabstop>checkFindBackward</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogFind</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogFind</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
